samples/kotlinDslExample/src/main/kotlin/HelloWorldServer.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2015, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.grpc.Server
+import io.grpc.ServerBuilder
+import io.grpc.examples.helloworld.GreeterGrpc
+import io.grpc.examples.helloworld.HelloReply
+import io.grpc.examples.helloworld.HelloRequest
+import io.grpc.stub.StreamObserver
+import java.io.IOException
+import java.util.logging.Logger
+
+/**
+ * Server that manages startup/shutdown of a `Greeter` server.
+ */
+class HelloWorldServer {
```

Comment:
This is an excellent example of writing using gRPC with kotlin. However, I think it's better suited for https://github.com/grpc/grpc-java so that this repo's examples do not assume gRPC.

Reply:
nevermind, I see now that the existing example project already includes gRPC in the build.gradle. I think we can include the gRPC kotlin app example as is.